docker-assets/initialize_httpd.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+#
+# Initial setup for httpd upon container startup.
+#
+
+# Define container environment variables needed by the httpd process
+#
+cat - <<!END! >> /etc/sysconfig/httpd
+
+#
+# Container Environment Variables needed
+#
+MANAGEIQ_SERVICE_NAME = "${MANAGEIQ_SERVICE_NAME}"
```

Comment:
I think this should be changed to something generic, and not MIQ specific, like `SERVICE_NAME` or `CONTAINER_SERVICE_NAME`